src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/MissingOverrideCheck.java

```diff
@@ -216,21 +218,20 @@ public void setJavaFiveCompatibility(final boolean compatibility) {
         return getRequiredTokens();
     }
 
+    @Override
+    public boolean isCommentNodesRequired() {
+        return true;
```

Comment:
https://gist.githubusercontent.com/strkkk/4dbcf7e137f836e68638c2422e540082/raw/65bca9217bebaa2eb8aa6361bb07daa0eae73aae/config2.xml
https://checkstyle.org/config_annotation.html#MissingOverride_Properties

Please provide one more regression run with `javaFiveCompatibility` turned on just to ensure new comment nodes has no issues anywhere.

Reply:
clean https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/0d09f3f_2022190021/reports/diff/index.html